src/main/java/com/bootcamp/demo/firebase/FirebaseController.java

```diff
@@ -30,18 +21,11 @@
 @RestController
 @RequestMapping(path = "/firebase", produces = APPLICATION_JSON_VALUE)
 public class FirebaseController {
```

Comment:
Would it be better to have all controller in the same package, to keep the layering?